frontend/src/board.html

```diff
@@ -0,0 +1,178 @@
+<!doctype html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8" />
+    <meta name="viewport" content="width=device-width, initial-scale=1" />
+    <title>Board</title>
+    <link href="css/board.css" rel="stylesheet" />
+    <style>
+      .chess-board {
+        width: 48rem;
+      }
+    </style>
+  </head>
+  <body>
+    <ul class="chess-board">
+      <li></li>
```

Comment:
why didn't you use a functions/Component for the many  li . Or why dont we use React?

Reply:
The problem is non-necessary overhead on the page load, which is an inconvenience on the users side.